2.0-PROPOSAL/README.md

```diff
@@ -0,0 +1,907 @@
+# Brigade 2.0 Proposal
+
+Since its first major release, Brigade has proven itself a useful and
+[moderately
+popular](https://www.cncf.io/wp-content/uploads/2020/03/CNCF_Survey_Report.pdf)
+platform for achieving _"event-driven scripting for Kubernetes."_ Over that same
+period, reports from our community and the maintainers' own efforts to "dog
+food" Brigade have exposed many issues and reasonable feature requests which can
+be addressed only by re-architecting the product and incurring some degree of
+breaking changes. _In other words, it is time to talk about Brigade 2.0._
+
+In this document, one project maintainer, operating with input from other
+maintainers, wishes to present a formal proposal for Brigade 2.0, and solicit
+feedback from and ratification by the broader Brigade community.
+
+## TL;DR
+
+Brigade 2.0 is proposed to introduce a less Kubernetes-centric experience,
+wherein users having little or no Kubernetes experience, or even those lacking
+direct access to a cluster, can quickly become productive. Breaking changes are
+on the docket, but Brigade 2.0 should feel familiar to anyone with previous
+Brigade experience. In short, the author proposes Brigade's nuanced transition
+from "event-driven scripting for Kubernetes" to "event driven scripting (for
```

Comment:
Until now, k8s has been at the very core of Brigade. Apart from being used as a makeshift doc store and makeshift message bus, it's also impossible to use Brigade without credentials for the k8s cluster and presumably no one gives you that sort of access unless you also have some k8s expertise. Some issues with the UX also prompt users to "drop down to k8s" for certain things. i.e. k8s is "in your face."

The main thrust of this proposal is about k8s fading into the background as an implementation detail-- albeit an important one.

Given that this appears in the tl;dr section, I think I might be ok with this statement being slightly less than crystal clear. Those who feel any confusion should read on, because all of the above is spelled out more clearly point by point.